dask_jobqueue/core.py

```diff
@@ -161,7 +167,8 @@ def job_file(self):
     def start_workers(self, n=1):
         """ Start workers and point them to our local scheduler """
         workers = []
-        for _ in range(n):
+        num_jobs = min(1, math.ceil(n / self.worker_processes))
+        for _ in range(num_jobs):
```

Comment:
This is a breaking change I want to make sure everyone is aware of. The current behavior for a hypothetical setup that includes 10 workers per job would be:

```Python
cluster.start_workers(1)
```

...and get 1 job and 10 workers. 

I'd like to change this so that `start_workers(n)` gives us `n` workers and as many jobs as needed to make that happen.

Reply:
Historically start_workers was a semi-convention between a few projects.  This has decayed, so I have no strong thoughts here.  I do think that we need to be consistent on `scale` though, which seems a bit more standard today.

Reply:
Will this really help adaptive? Would'nt there still be a problem with starting the worker in a grouped manner?

With your example, calling `cluster.start_workers(1)` will still lead to 1 job and 10 workers!

But this may be well handled by adaptive, I don't know. In this case, this may not be needed to do this breaking change?